api/services/tournamentService.js

```diff
@@ -64,7 +65,7 @@ const createTournament = async (
     endDate,
     location
 ) => {
-    const user = await userModel.getUserByFirebaseId(uid);
+    const user = await UserModel.getUserByFirebaseId(uid);
     if (
         user.role !== "Super Admin" &&
```

Comment:
This works, if you feel like making a 1-line change 
```suggestion
        user.role !== UserModel.Roles.ADMIN &&
```